phalcon/Config.zep

```diff
@@ -39,7 +39,7 @@ use Phalcon\Config\Exception;
  * );
  *</code>
  */
```

Comment:
Can we change `pathDelimiter` to a non static property (along with relevant calls and methods). No need to have it like that. `setPathDelimiter` can return self also so that will help with chaining